api/proto/teleport/legacy/types/types.proto

```diff
@@ -944,6 +944,20 @@ message AppV3 {
   ];
 }
 
+// CORS defines the CORS policy for AppSpecV3
+message CORS {
+  // AllowedOrigins specifies which origins are allowed to access the app.
+  repeated string AllowedOrigins = 1 [(gogoproto.jsontag) = "allowed_origins,omitempty"];
```

Comment:
Can we use the new proto style rules with names using snake case instead of camel?

Reply:
You mean like this?
```proto
repeated string allowed_origins = 1 [(gogoproto.jsontag) = "allowed_origins,omitempty"];
```

Reply:
Yes.